ruff.toml

```diff
@@ -0,0 +1,27 @@
+exclude = ["dist","build","venv",".venv",".tox",".git"]
+line-length = 88
+target-version = "py38"
+
+[lint]
+ignore = [
+    "E501", # line too long
+    "S101", # flake8-bandit Use of assert detected
```

Comment:
it may be better to move this to https://docs.astral.sh/ruff/settings/#lint_extend-per-file-ignores and limit it to just test files.

https://github.com/wagtail/wagtail-factories/blob/b4d53cf357b6a1b96e5df68a56a79732ad7965ba/src/wagtail_factories/factories.py#L77 would be better with a proper exception

Reply:
Nice suggestion, fixed in 3dba08e